quick-start/src/main/ui/app/header/header.component.html

```diff
@@ -12,7 +12,7 @@
     <div class="mdl-layout-spacer"></div>
     <nav class="mdl-navigation mdl-layout--large-screen-only">
       <a id="database-tab" class="mdl-navigation__link" [ngClass]="{'active': isActive('/')}" [routerLink]="['']"><i class="fa fa-dashboard"></i> Dashboard</a>
-      <a id="entities-tab" class="mdl-navigation__link" *ngIf="getMarkLogicVersion().startsWith('9')" [ngClass]="{'active': isActive('/entities')}" [routerLink]="['entities']"><i class="fa fa-industry"></i> Entities</a>
+      <a id="entities-tab" class="mdl-navigation__link" *ngIf="!getMarkLogicVersion().startsWith('8')" [ngClass]="{'active': isActive('/entities')}" [routerLink]="['entities']"><i class="fa fa-industry"></i> Entities</a>
```

Comment:
I agree with @popzip that we should be >= to 9, not just looking for 'not 8' for completeness of version checking.

Reply:
@aebadirad, can you tell me what ML version formats I can expect from the /api/current-project endpoint? I know that value looks like "10.0-20180115" on my local build but I'm not sure how/if it varies with other distributions.

Reply:
i think to get the major version you can always get the value before the first period and parse and an integer.

Reply:
@aebadirad I addressed the review comments with https://github.com/marklogic-community/marklogic-data-hub/pull/646/commits/05407a3facd4a0cc80a75b0f973ba71469327656. Can we merge this?